eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -229,6 +231,11 @@ jobs:
           value: real
         - name: signArguments
           value: -sign /p:DotNetSignType=real /p:TeamName=$(_TeamName)
+        - name: _EnableDacSigning
+          ${{ if and(eq(parameters.isBuiltFromVmr, true), ne(parameters.buildSourceOnly, 'True')) }}:
```

Comment:
Should this have a "only run in official builds" condition as well?

Reply:
We need this for all BuildPass1 Windows verticals as well (the non-cross-os DACs need to be signed as well).